src/main/java/de/uhd/ifi/se/decision/management/jira/rest/ConfigRest.java

```diff
@@ -72,6 +72,9 @@ private Response checkRequest(HttpServletRequest request, String projectKey, Str
 		if (isActivatedString == null) {
 			return Response.status(Status.BAD_REQUEST).entity(ImmutableMap.of("error", "isActivated = null")).build();
 		}
+		if (isActivatedString != "true" && isActivatedString != "false"){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use equals() to compare strings instead of '==' or '!='](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5455522)